tests/test_coroutine.rs

```diff
@@ -0,0 +1,98 @@
+#![cfg(feature = "macros")]
+#![cfg(not(target_arch = "wasm32"))]
```

Comment:
I've added this line to deal with *emscripten* for now. Is it ok for you? Can I let you open the issue? I don't really know anything about wasm.